src/main/java/org/broadinstitute/dsm/util/DDPKitRequest.java

```diff
@@ -109,7 +94,26 @@ public void requestAndWriteKitRequests(List<LatestKitRequest> latestKitRequests)
                                                 }
                                             }
                                             else {
-                                                throw new RuntimeException("No participant_index was setup for  " + latestKit.getInstanceName());
+                                                //kit requests from gen2 can be removed after all studies are migrated
```

Comment:
this section can be removed, because we do not have any gen2 studies with automated kits anymore

Reply:
Done!